profile/src/main/java/org/apache/karaf/profile/assembly/ConfigInstaller.java

```diff
@@ -93,7 +99,54 @@ public void installConfigs(Feature feature, Downloader downloader, ArtifactInsta
                     }
                 }
             }
+            for (ConfigFile configFile : content.getConfigfile()) {
+                if (pidMatching(FilenameUtils.getBaseName(configFile.getFinalname()))) {
+                    installConfig(downloader, configFile);
+                }
+            }
+        }
+    }
+
+    private void installConfig(Downloader downloader, ConfigFile pConfigFile) throws Exception {
```

Comment:
The method should be named `installConfigFile` to  avoid confusion.

Reply:
Correct.